hummingbird/ml/_container.py

```diff
@@ -112,11 +118,17 @@ def _run_batch_inference(self, function, *inputs, reshape=False):
             # Tell function that we are in the last iteration and do proper actions in case
             # (e.g., for TVM we may want to use the raminder model).
             self._last_iteration = i == iterations - 1
-            predictions.extend(function(*batch).ravel())
+            out = function(*batch)
+            out = np.expand_dims(out, 1) if len(out.shape) == 1 else out
+
+            if predictions is None:
```

Comment:
I had to add this check for the case when `TEST_INPUT` is not provided (`torch` backend), but I realized that if I have this I don't need `num_output_columns` or `output_index` changes...